plugins/org.jboss.tools.openshift.client/pom.xml

```diff
@@ -28,7 +28,7 @@
 		<commons-codec.version>1.6</commons-codec.version>
 		<commons-lang.version>2.6</commons-lang.version>
 		<commons-io.version>2.1</commons-io.version>
-		<enforcer.skip>true</enforcer.skip>
+		<!-- <enforcer.skip>true</enforcer.skip> -->
```

Comment:
had to comment it when we started the current sprint using a SNAPSHOT client library. The build would have failed constantly on the enforcer rule. Now that the switch to the final version of the client we can get the enforcer back. But I ack that the current approach is kinda useless.